DSC/dsc.py

```diff
@@ -471,13 +481,15 @@ def get_host_base_from_uri(blob_uri):
     return netloc[netloc.find('.'):]
 
 def download_external_file(file_uri, download_dir):
+    waagent.AddExtensionEvent(name=ExtensionShortName, op="EnableInProgress", isSuccess=True, message="Downloading from external file")
```

Comment:
Currently, we are getting exceptions but generic ones. We need more logging and concrete exceptions to debug issues with Enable